command_test.go

```diff
@@ -2,10 +2,15 @@ package cmds
 
 import (
 	"context"
+	"errors"
 	"io"
 	"testing"
 	"time"
 
+	"sync"
```

Comment:
We generally group like imports together. Standard, then local (this repo), then remote (other repos) with newlines between).